updatecli/weekly.d/contributing.yaml

```diff
@@ -0,0 +1,55 @@
+name: Update Technical Requirements in CONTRIBUTING.md
+
+sources:
+  getMavenLabel:
+    kind: file
+    name: Get Maven label from Jenkinsfile
+    spec:
+      file: Jenkinsfile
+      marker: 'label '''
+      pattern: '(maven-[0-9]+)'
```

Comment:
```suggestion
```

here the maven label seem worthless, either you use it to match the agent definition in common.yaml or you just consider that it will always be the all-in-one, which seems fair to me.